tests/integ/timeout.py

```diff
@@ -59,6 +59,18 @@ def handler(signum, frame):
         signal.alarm(0)
 
 
+@contextmanager
+def timeout_training(seconds=0, minutes=20, hours=0):
```

Comment:
If the purpose of these two wrappers is to set a default timeout period I would just do something like this:

```
TRAINING_JOB_DEFAULT_TIMEOUT_MINUTES = 20

with timeout(minutes=TRAINING_JOB_DEFAULT_TIMEOUT_MINUTES)
```

Reply:
Updated! Thanks!